packages/test-utils/src/server.ts

```diff
@@ -4,22 +4,30 @@ import { getRandomPort, waitForPort } from 'get-port-please'
 import { fetch as _fetch, $fetch as _$fetch, FetchOptions } from 'ohmyfetch'
 import { useTestContext } from './context'
 
-export async function listen () {
+export async function startServer () {
   const ctx = useTestContext()
   const port = await getRandomPort()
   ctx.url = 'http://localhost:' + port
+  stopServer()
   ctx.serverProcess = execa('node', [
     // @ts-ignore
     resolve(ctx.nuxt.options.nitro.output.dir, 'server/index.mjs')
   ], {
     env: {
+      ...process.env,
       PORT: String(port),
       NODE_ENV: 'test'
     }
   })
   await waitForPort(port, { retries: 8 })
 }
 
+export async function stopServer (ctx = useTestContext()) {
```

Comment:
Refactor for consitancy: 
```suggestion
export async function stopServer () {
ctx = useTestContext()
```

Reply:
this is because it's likely common to run in an `afterAll` or `afterEach` hook where the context has been unset

Reply:
this code will fail as is...